src/dev_genesis_wasm.erl

```diff
@@ -61,17 +61,24 @@ ensure_started(Opts) ->
     % of the registered name implies its availability.
     {ok, Cwd} = file:get_cwd(),
     ?event({ensure_started, cwd, Cwd}),
-    IsDevelopment = string:str(Cwd, "rel/hb"),
+    BinDir = filename:join([Cwd, "bin"]),
```

Comment:
Was this intended to be in this PR? Please can you remove it and work on that separately?